examples/s3/README.md

```diff
@@ -15,9 +16,32 @@ resources:
     class: basic-admin
 ```
 
+## Infrastructure setup
+The workload service account will be automatically assigned to the necessary AWS IAM Role with the selected IAM Policy.
+
+```mermaid
+graph TD;
+  s3["Amazon S3 bucket"]
+  policy["Amazon IAM Policy"]
```

Comment:
Should the policy also be pointed at `s3` to signal that policy grants access to this bucket?

Reply:
Good idea! Added